content/en/docs/concepts/glossary.md

```diff
@@ -1,100 +1,108 @@
 weight: 200
 ---
 
-The OpenTelemetry project uses terminology you might not be familiar with. In
-addition, the project defines the terminology in a different way than others.
-This page captures terminology used in the project and what it means.
+This glossary defines terms and [concepts](/docs/concepts/) that are new to the
+OpenTelemetry project, and clarifies OpenTelemetry-specific uses of terms common
+in the telemetry field.
 
-## Generic terminology
+We also comment on spelling and capitalization when helpful. For example, see
+[OpenTelemetry](#opentelemetry) and [OTel](#otel).
 
-### **Aggregation**
+## Terms
+
+### Aggregation
 
 The process of combining multiple measurements into exact or estimated
 statistics about the measurements that took place during an interval of time,
 during program execution. Used by the [Metric](#metric)
 [Data source](#data-source).
 
-### **API**
+### API
 
 Application Programming Interface. In the OpenTelemetry project, used to define
 how telemetry data is generated per [Data source](#data-source).
 
-### **Application**
+### Application
 
 One or more [Services](#service) designed for end users or other applications.
 
-### **APM**
+### APM
 
 Application Performance Monitoring is about monitoring software applications,
 their performance (speed, reliability, availability, and so on) to detect
 issues, alerting and tooling for finding the root cause.
 
-### **Attribute**
+### Attribute
 
 OpenTelemetry term for [Metadata](#metadata). Adds key-value information to the
 entity producing telemetry. Used across [Signals](#signal) and
 [Resources](#resource). See [attribute spec][attribute].
 
-### **Automatic instrumentation**
+### Automatic instrumentation
 
 Refers to telemetry collection methods that do not require the end-user to
 modify application's source code. Methods vary by programming language, and
 examples include bytecode injection or monkey patching.
 
-### **Baggage**
+### Baggage
 
 A mechanism for propagating [Metadata](#metadata) to help establish a causal
 relationship between events and services. See [baggage spec][baggage].
 
-### **Client library**
+### Client library
 
 See [Instrumented library](#instrumented-library).
 
-### **Client-side app**
+### Client-side app
 
 A component of an [Application](#application) that is not running inside a
 private infrastructure and is typically used directly by end-users. Examples of
 client-side apps are browser apps, mobile apps, and apps running on IoT devices.
 
-### **Collector**
+### Collector
+
+The [OpenTelemetry Collector], or Collector for short, is a vendor-agnostic
+implementation on how to receive, process, and export telemetry data. A single
+binary that can be deployed as an agent or gateway.
 
-A vendor-agnostic implementation on how to receive, process, and export
-telemetry data. A single binary that can be deployed as an agent or gateway.
+Also known as.
```

Comment:
I'm not sure what this is referring to.

Reply:
Oops, that's meant to be deleted. I'll remove it.